.generated_files

```diff
@@ -0,0 +1,2 @@
+api_docs/*
+src/main/java/com/datadog/api/*
```

Comment:
is it glob or regex?

Reply:
Its regex - https://github.com/DataDog/apigentools/blob/master/apigentools/commands/generate.py#L426
Fixed :)